benchmarking/commons/default_baselines.py

```diff
@@ -0,0 +1,130 @@
+# Copyright 2021 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License").
+# You may not use this file except in compliance with the License.
+# A copy of the License is located at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# or in the "license" file accompanying this file. This file is distributed
+# on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+# express or implied. See the License for the specific language governing
+# permissions and limitations under the License.
+from typing import Dict, Any
+
+from benchmarking.commons.baselines import (
+    search_options,
+    default_arguments,
+    MethodArguments,
+)
+from syne_tune.optimizer.baselines import (
+    RandomSearch as _RandomSearch,
+    GridSearch as _GridSearch,
+    BayesianOptimization as _BayesianOptimization,
+    KDE as _KDE,
+    BORE as _BORE,
+    BoTorch as _BoTorch,
+    REA as _REA,
+    ConstrainedBayesianOptimization as _ConstrainedBayesianOptimization,
+    ASHA as _ASHA,
+    MOBSTER as _MOBSTER,
+    HyperTune as _HyperTune,
+    BOHB as _BOHB,
+    ASHABORE as _ASHABORE,
+    SyncHyperband as _SyncHyperband,
+    SyncBOHB as _SyncBOHB,
+    DEHB as _DEHB,
+    SyncMOBSTER as _SyncMOBSTER,
+)
+from syne_tune.util import recursive_merge
+
+
+def _baseline_kwargs(
+    method_arguments: MethodArguments,
+    kwargs: Dict[str, Any],
+    is_multifid: bool = False,
+) -> Dict[str, Any]:
+    da_input = dict(
+        config_space=method_arguments.config_space,
+        search_options=search_options(method_arguments),
+    )
+    if is_multifid:
+        da_input["resource_attr"] = method_arguments.resource_attr
+    result = recursive_merge(
+        default_arguments(method_arguments, da_input),
+        kwargs,
+        stop_keys=["config_space"],
+    )
+    return result
+
+
+def RandomSearch(method_arguments: MethodArguments, **kwargs):
+    return _RandomSearch(**_baseline_kwargs(method_arguments, kwargs))
+
```

Comment:
@this and all other definitions: Thats really cool and makes defining new benchmarks much easier!

Reply:
Yes, now you only need to import default_baselines, and then you can use it almost like optimizer/baselines, but all the arguments are done behind the scenes.